src/pages/AnimeFenix/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "urls": {
```

Comment:
Search url is missing

Example: https://github.com/MALSync/MALSync/blob/master/src/pages/AnimeXin/meta.json#L2